cetmix_tower_server/models/cx_tower_plan.py

```
@@ -263,6 +263,13 @@ def _run_single(self, server, **kwargs):
                 )
                 return plan_log
 
+        # Commit the current transaction before starting the plan.
+        # This ensures that changes (e.g. preventing parallel runs) are
+        # immediately visible to other transactions. Without this, the same
+        # plan could be started again on the same server by another user
+        # before the transaction is committed.
+        self.env.cr.commit()  # pylint: disable=E8102
```

Pylint (3.3.7)

[warning] Unknown option value for 'disable', expected a valid pylint message and got 'E8102'

(W0012)